docs/guide/tutorial-docker.md

```diff
@@ -100,4 +100,4 @@ When running MySQL as (`mysql`), you can add for example a PHPmyAdmin container
             - PMA_HOST=mysql
         depends_on:
             - mysql
```

Comment:
I think we should give a snippet for mysql

```
    mysql:
        image: ${MYSQL_IMAGE_NAME}
        container_name: ${MYSQL_HOST}
        environment:
            - MYSQL_DATABASE=${MYSQL_DATABASE}
            - MYSQL_ROOT_PASSWORD=${MYSQL_ROOT_PASSWORD}
            - MYSQL_USER=${MYSQL_USER}
            - MYSQL_PASSWORD=${MYSQL_PASSWORD}
        ports:
            - '3306:3306'
        volumes:
            - ./data/db/mysql:/var/lib/mysql
```

Reply:
@samdark Should I add some more db examples here or in another PR?
@FabrizioCaldarelli btw: volumes for MySQL as shown can be problematic on several setups, due to permission issues - just saying :)

Reply:
@schmunk42 We can remove volumes confg.

Reply:
@schmunk42 here is better.